CHANGES

```diff
@@ -53,12 +53,16 @@
      simplified testing OpenSSL::Test::Simple.
 
      For documentation on our testing modules, do:
-
         perldoc test/testlib/OpenSSL/Test/Simple.pm
         perldoc test/testlib/OpenSSL/Test.pm
-
      [Richard Levitte]
```

Comment:
sure, okay fine.  but every other entry has no blank line before the [credit] :)